inc/migrate-terms-data.php

```diff
@@ -0,0 +1,288 @@
+<?php
+/**
+ * Data for terms migration
+ */
+
+/**
+ * Get data for transforming WP terms to Hugo terms
+ *
+ * @param int|string $term_id WordPress term ID, integer or numeric string
+ * @return array|null Term data with Hugo taxonomy and term strings, or null if not found
+ */
+function hh_hugo_get_term_data( $term_id ) {
+	$terms_data = array(
```

Comment:
I'd love to see this defined as a constant so that it isn't redefined every time the function is called.